tables-book/04-02-events.Rmd

```diff
@@ -277,6 +277,45 @@ ft <- labelizor(
 ft
 ```
 
+
+#### tfrmt
+
+```{r}
+resetSession()
+library(tfrmt)
+library(dplyr)
+
+# Get data
+data_ae <- tfrmt::data_ae
```

Comment:
As above, this is different data than everyone else is using. Please use the same data the other sections use.

Reply:
Completed here: https://github.com/RConsortium/rtrs-wg/pull/100